Source/Core/VideoBackends/Vulkan/Renderer.h

```diff
@@ -87,62 +87,13 @@ class Renderer : public ::Renderer
-                 const TargetRectangle& scaled_efb_rect, u32 xfb_addr,
-                 const XFBSourceBase* const* xfb_sources, u32 xfb_count, u32 fb_width,
-                 u32 fb_stride, u32 fb_height);
-  void DrawEFB(VkRenderPass render_pass, const TargetRectangle& target_rect,
-               const TargetRectangle& scaled_efb_rect);
-  void DrawVirtualXFB(VkRenderPass render_pass, const TargetRectangle& target_rect, u32 xfb_addr,
-                      const XFBSourceBase* const* xfb_sources, u32 xfb_count, u32 fb_width,
-                      u32 fb_stride, u32 fb_height);
-  void DrawRealXFB(VkRenderPass render_pass, const TargetRectangle& target_rect,
-                   const XFBSourceBase* const* xfb_sources, u32 xfb_count, u32 fb_width,
-                   u32 fb_stride, u32 fb_height);
-
   // Draw the frame, as well as the OSD to the swap chain.
-  void DrawScreen(const TargetRectangle& scaled_efb_rect, u32 xfb_addr,
-                  const XFBSourceBase* const* xfb_sources, u32 xfb_count, u32 fb_width,
-                  u32 fb_stride, u32 fb_height);
-
-  // Draw the frame only to the screenshot buffer.
-  bool DrawFrameDump(const TargetRectangle& scaled_efb_rect, u32 xfb_addr,
-                     const XFBSourceBase* const* xfb_sources, u32 xfb_count, u32 fb_width,
-                     u32 fb_stride, u32 fb_height, u64 ticks);
-
-  // Sets up renderer state to permit framedumping.
-  // Ideally we would have EndFrameDumping be a virtual method of Renderer, but due to various
-  // design issues it would have to end up being called in the destructor, which won't work.
-  void StartFrameDumping();
-  void EndFrameDumping();
-
-  // Fence callback so that we know when frames are ready to be written to the dump.
-  // This is done by clearing the fence pointer, so WriteFrameDumpFrame doesn't have to wait.
-  void OnFrameDumpImageReady(VkFence fence);
-
-  // Writes the specified buffered frame to the frame dump.
-  // NOTE: Assumes that frame.ticks and frame.pending are valid.
-  void WriteFrameDumpImage(size_t index);
-
-  // If there is a pending frame in this buffer, writes it to the frame dump.
-  // Ensures that the specified readback buffer meets the size requirements of the current frame.
-  StagingTexture2D* PrepareFrameDumpImage(u32 width, u32 height, u64 ticks);
-
-  // Ensures all buffered frames are written to frame dump.
-  void FlushFrameDump();
+  void DrawScreen(VKTexture* xfb_texture, float horizontal_scaling);
 
   // Copies/scales an image to the currently-bound framebuffer.
   void BlitScreen(VkRenderPass render_pass, const TargetRectangle& dst_rect,
-                  const TargetRectangle& src_rect, const Texture2D* src_tex);
-
-  bool ResizeFrameDumpBuffer(u32 new_width, u32 new_height);
-  void DestroyFrameDumpResources();
+                  const TargetRectangle& src_rect, const Texture2D* src_tex,
+                  BlendingState& blend_state);
```

Comment:
This can be a const reference as far as I can tell (in conjunction with my previous comment about constness).